packages/aws-cdk/lib/os.ts

```diff
@@ -96,3 +98,19 @@ function windowsEscape(x: string): string {
   const shellMeta = new Set<string>(['"', '&', '^', '%']);
   return x.split('').map(c => shellMeta.has(x) ? '^' + c : c).join('');
 }
+
+/**
+ * Return the CDK_HOME path, or ~/.cdk
+ */
+export function getCdkHome(cacheDirectory = true): string {
```

Comment:
remove the parameter. make this just return the home directory and add the `/.cdk` suffix outside.